### PR TITLE
Improve usage output of deployment script

### PR DESCRIPTION
**Pull request by ctron**
This change shows an explicit list of options on cases where options are available and also always falls back to showing the usage help when an argument is unknown.

### CODE REVIEW

File: templates/install/deploy.sh [hunk 1]
```diff
@@ -66,28 +66,27 @@ while getopts a:de:gm:n:o:u:t:yvh opt; do
...
+        *)
             echo "usage: deploy.sh [options]"
             echo
             echo "deploy the EnMasse suite into a running Kubernetes cluster"
             echo
             echo "optional arguments:"
             echo "  -h                   show this help message"
-            echo "  -a \"none standard\"   Deploy given authentication services (default: \"none\")"
+            echo "  -a TYPE              Deploy given authentication services (default: \"none\")"
+            echo "                         TYPE := { none | standard }"
```
Comment (lulf) [inline templates/install/deploy.sh]: It is possible to deploy both, should there by a different syntax for that?

Comment (ctron) [reply templates/install/deploy.sh]: Oh I see .. yes. You are right I will change that.

**Comment by ctron** [comment]
So this is how e.g. `ss` does it:
~~~
   -A, --query=QUERY, --socket=QUERY
       QUERY := {all|inet|tcp|udp|raw|unix|unix_dgram|unix_stream|unix_seqpacket|packet|netlink|vsock_stream|vsock_dgram}[,QUERY]
~~~

However that requires a comma separated list. That is why I needed to change the for loop in addition.

**Comment by lulf** [comment]
LGTM!

**Pull request merged**